src/main/java/org/jenkinsci/backend/ircbot/IrcListener.java

```diff
@@ -750,19 +756,14 @@ private boolean addGitHubCommitter(Channel channel, User sender, String collabor
             GitHub github = GitHub.connect();
             GHUser c = github.getUser(collaborator);
             GHOrganization o = github.getOrganization(IrcBotConfig.GITHUB_ORGANIZATION);
-            
+
             final GHTeam t;
                 GHRepository forThisRepo = o.getRepository(justForThisRepo);
                  if (forThisRepo == null) {
                      out.message("Could not find repository:  "+justForThisRepo);
                      return false;
                  }
                  t = getOrCreateRepoLocalTeam(o, forThisRepo);
-                
```

Comment:
From looking at this diff, (I may be missing context when viewing), I am wondering why this was removed. Is `t` guaranteed to never be null?

Reply:
Yes, it will either create the team or find the existing one